src/Agoda.IoC.Core/ContainerRegistrationExceptionContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Agoda.IoC.Core
+{
+    public class ContainerRegistrationContextException : Exception
+    {
+        public ContainerRegistrationContextException(string message): base(message)
+        {
+        }
+        public bool Ignore { get; set; }
+        public List<RegistrationContextException> RegistrationContextExceptions { get; } 
```

Comment:
Since Agoda.Analyzers is referred in this project, please be aware that this will probably introduce a new buildtime warning.

Reply:
will fix this